reports/Illinois.md

```diff
@@ -191,6 +191,22 @@ id: il-chicago-14
 * https://twitter.com/punishmegrant/status/1268966252707487745
 
 
+### Cop sucker punches activist | July 17th
```

Comment:
I don't think sucker punch is the most neutral description. Just punch would be better imo

Reply:
Changed to just 'punch'